doc/src/tutorial/intro.rst

```diff
@@ -6,33 +6,33 @@ What is Symbolic Computation?
 =============================
 
 Symbolic computation deals with the computation of mathematical objects
-symbolically.  This means that the mathematical objects are represented
+symbolically. This means that the mathematical objects are represented
 exactly, not approximately, and mathematical expressions with unevaluated
 variables are left in symbolic form.
 
 Let's take an example. Say we wanted to use the built-in Python functions to
-compute square roots. We might do something like this
+compute square roots. We might do something like this.
 
    >>> import math
    >>> math.sqrt(9)
    3.0
 
 9 is a perfect square, so we got the exact answer, 3. But suppose we computed
-the square root of a number that isn't a perfect square
+the square root of a number that isn't a perfect square.
 
    >>> math.sqrt(8)
    2.82842712475
 
 Here we got an approximate result. 2.82842712475 is not the exact square root
```

Comment:
I reverted the change here. Two complete sentences shouldn't be joined by comma.